src/lib/hooks/use-floating.svelte.ts

```diff
@@ -0,0 +1,123 @@
+import {
```

Comment:
Consider linking the Floating UI equivalent docs and source at the top of each hook or component.